docs/imviz/displayimages.rst

```diff
@@ -145,6 +145,36 @@ You can :ref:`import <imviz-import-regions-api>` and :ref:`export <imviz_export_
 There are options available in the :guilabel:`Layer` tab under the |icon-settings-sliders| icon
 to make subsets visible or invisible, to change their color, and to change their opacity.
 
+.. _imviz_defining_spatial_regions_single_pixel:
+
+Single-Pixel Selection
+----------------------
+
+This tool allows the user to create a single-pixel spatial region
+in an image viewer. Activate this tool and then left-click to create
+the new region. Click again to move it to a new location under the cursor.
+Users can hold down the alt key (Alt key on Windows, Option key on Mac)
+while clicking on a pixel to create a new region at that point instead
+of moving it. Users can also utilize the different subset modes that are
+explained in the :ref:`Spatial Regions <imviz_defining_spatial_regions>`
+section above.
+
+When you have multiple images loaded and linked by WCS
+(see :ref:`imviz-link-control`), the region defined is with respect to
+the reference image, which might not be the image you are viewing.
+This behavior is consistent with glue.
```

Comment:
If we reference Glue we should probably capitalize it and link to the Glue docs (or the section of our docs explaining Glue). But then again - does the average user care that this is consistent with Glue? I think most people would find this comment cryptic.

Reply:
@astrofrog insistent that Glue should be spelled with lowercase. See https://github.com/spacetelescope/jdaviz/pull/1448 🤷

Reply:
```suggestion
```

Oh, good to know. Either way, the more I think about it the more I don't think it's useful information for the user, so I'm going to suggest removing that line. I'm ok with being overruled though, so I'll approve in a moment.

Reply:
That's fine. I removed it. Thanks!